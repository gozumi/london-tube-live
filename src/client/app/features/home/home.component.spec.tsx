import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import * as React from 'react'

import Home from './home.component'

describe('feature: Home', () => {
  it('should render correctly', () => {
    const home = shallow(<Home />)
    expect(shallowToJson(home)).toMatchSnapshot()
  })
})
