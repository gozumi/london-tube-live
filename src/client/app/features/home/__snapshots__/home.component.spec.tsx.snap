// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`feature: Home should render correctly 1`] = `
<section
  className="home"
>
  <Connect(JourneyContainer) />
</section>
`;
